recipes/lua/all/conanfile.py

```diff
@@ -22,6 +22,9 @@ def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         extracted_dir = "lua-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
+        if self.settings.os == 'Android' or self.settings.os == 'iOS':
```

Comment:
Is it necessary to patch only for these platforms? Would it break for others?

Reply:
Yeah, it would be nice applying patches which for any platform. You can use #ifdef for instance

Reply:
yes, only for mobile, since there system calls are not wanted and disabled

Reply:
I think your patch should be applied for any platform. It has a ifdef to protect.

Reply:
ah now I understand what you mean! 
yes, of course we added those include guard, so it can be applied for all builds, 
will do that. Thanks for the hint!

Reply:
done